crates/http-api-bindings/src/embedding/llama.rs

```diff
@@ -37,19 +41,39 @@ struct EmbeddingResponse {
 #[async_trait]
 impl Embedding for LlamaCppEngine {
     async fn embed(&self, prompt: &str) -> anyhow::Result<Vec<f32>> {
-        let request = EmbeddingRequest {
-            content: prompt.to_owned(),
-        };
+        // Some initial requests to llama.cpp are experiencing issues
```

Comment:
```suggestion
        // Some initial requests to llama.cpp are experiencing issues
```

Please refine the docs here to reflect the fact that not only "initial request", but from time to time whenever embedding server being vacant for a while.